osclib/core.py

```diff
@@ -56,10 +56,52 @@ def groups_members(apiurl, groups):
 
     return members
 
-# osc uses xml.etree while we rely on lxml
+
+def get_request_list_with_history(
```

Comment:
Can you add a deprecation warning or at least a `# TODO` comment?

Reply:
there's one on the line below (63). I can make it a deprecation warning when everything else is green.

Reply:
AFAIK a warning wouldn't make it less green than it was previously, wouldn't it?

Reply:
It does not. just want to get to see that everything works before adding further code changes (that would restart the CI). do want to leave from work today at some point soon ;)

Reply:
Ah, so it's just about the push, I see. In that case we'll need a separate PR anyway.

> do want to leave from work today at some point soon ;)

Protip: You can do that while the CI is running - /me writes this reply from home :stuck_out_tongue: